Sort/ArrayPartitionI.swift

```diff
@@ -0,0 +1,13 @@
+class Solution {
+    func arrayPairSum(_ nums: [Int]) -> Int {
+        var arr = nums
+        arr = arr.sorted { $0 < $1 }
+        var res = 0
+        for i in 0 ..< arr.count {
+            if i & 1 != 0 {
+                res += min(arr[i], arr[i - 1])
+            }
+        }
+        return res
```

Comment:
All 10 lines could be replaced by 1 line: return nums.sorted(by: <).enumerated().flatMap { $0 % 2 == 0 ? $1 : nil }.reduce(0) { $0 + $1 }